src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -2,6 +2,14 @@
 
 Release History
 ===============
+Unreleased
+++++++++++++++++++
+BC:webapp: fix inconsistencies in the output of "az webapp config appsettings delete/set"
```

Comment:
Can you make in 'Breaking Change' instead of 'BC'?
Docs team prefer to spell it out